src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/TdsEnums.cs

```diff
@@ -1206,8 +1208,11 @@ public enum SqlAuthenticationMethod
         /// <include file='../../../../../../doc/snippets/Microsoft.Data.SqlClient/SqlAuthenticationMethod.xml' path='docs/members[@name="SqlAuthenticationMethod"]/ActiveDirectoryDefault/*'/>
         ActiveDirectoryDefault,
 #if ADONET_CERT_AUTH && NETFRAMEWORK
-        SqlCertificate
+        SqlCertificate,
 #endif
+
+        /// <include file='../../../../../../doc/snippets/Microsoft.Data.SqlClient/SqlAuthenticationMethod.xml' path='docs/members[@name="SqlAuthenticationMethod"]/ActiveDirectoryWorkloadIdentity/*'/>
```

Comment:
This comment is a bit hard to write because it may be the 10th or 11th value. I could make this doc inclusion conditional?

Reply:
You can include it above SqlCertificate, does that work?

Reply:
Yeah, I noticed that was done previously too, so no problem.